### ISSUE TITLE
[form-checkbox & radio] Investigate Firefox Screenshots

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-core#2444]
# Test Report

## Description
Need to investigate bad firefox screenshots for terra-form-checkbox. This does not appear to be an issue for chrome or internet explorer.

## Additional Context / Screenshots
![Screen Shot 2019-06-10 at 1 33 23 PM](https://user-images.githubusercontent.com/14099737/59218211-4a999080-8b85-11e9-89ce-7c868c0e98af.png)
![Screen Shot 2019-06-10 at 1 33 31 PM](https://user-images.githubusercontent.com/14099737/59218212-4a999080-8b85-11e9-8bc3-34339e5907b2.png)
![Screen Shot 2019-06-10 at 1 33 38 PM](https://user-images.githubusercontent.com/14099737/59218213-4b322700-8b85-11e9-96cf-b8f4c07b6242.png)

## Environment

* Component Name and Version: terra-form-checkbox v3.10.0 & terra-form-radio v3.12.0
* Browser Name and Version: selenium-firefox node 3.14-helium => Mozilla Firefox: 62.0.3 &
GeckoDriver: 0.23.0
* Testing against: https://github.com/cerner/terra-toolkit/tree/some_new_updates

## @ Mentions
@ryanthemanuel @mjhenkes

**Comment by yuderekyu** [comment]
:(

**Comment by emilyrohrbough** [comment]
It might be worth reviewing the last reference screenshot generation from [this PR](https://github.com/cerner/terra-core/pull/2416). I had noted I was unsure of the new screenshots in [this commit](https://github.com/cerner/terra-core/pull/2416/commits/24d6abb8cefc51e8e330c20b984ecd74668b3d6e?short_path=0e9881d#diff-0e9881d4280e2694b6f6cb8a33dc7c90)

**Comment by emilyrohrbough** [comment]
AHA! have located the problem. This is an known issue with taking screenshots of form elements with headless firefox:
- https://bugzilla.mozilla.org/show_bug.cgi?id=1385711
- https://github.com/mozilla/geckodriver/issues/1358

I have removed the headless configuration and screenshots appear as expected. I have also validated that each terra test viewport is reached with success (this was an issue form chrome, which is why chrome tests now run headless).
